Solutions/Azure SQL Database solution for sentinel/Hunting Queries/HuntingQuery-BooleanBlindSQLi.yaml

```diff
@@ -54,7 +54,7 @@ query: |
   | union quoteData
   | where left == right
   | extend alertText = strcat(left, "=", right)
-  | summarize AlertText=make_list(alertText, 10000) by TimeGenerated, Database, ClientIp, PrincipalName, Statement, ApplicationName, ResourceId
+  | summarize AlertText=make_list(alertText, 10000) by TimeGenerated, Database, ClientIp, PrincipalName, Statement, ApplicationName, ResourceId, HostName
```

Comment:
Please check with Manish on this query if this change is required or not.

Reply:
As I can see that Query has hostname mapping, but it's not visible in result, so adding hostname in result is required.